specification/devspaces/resource-manager/Microsoft.DevSpaces/stable/2019-04-01/devspaces.json

```diff
@@ -701,6 +706,9 @@
     },
     "OrchestratorSpecificConnectionDetails": {
       "description": "Base class for types that supply values used to connect to container orchestrators",
+      "required": [
+        "instanceType"
+      ],
```

Comment:
this is breaking change.. 
[https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/Doc.aspx?sourcedoc=%7B66505FCE-8057-4524-B84F-4CEFE2D8A77C%7D&file=ARMResourcesWithStrongIdentity.docx&action=default&mobileredirect=true](https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/Doc.aspx?sourcedoc=%7B66505FCE-8057-4524-B84F-4CEFE2D8A77C%7D&file=ARMResourcesWithStrongIdentity.docx&action=default&mobileredirect=true)

Reply:
I'll revert this. This change is not needed.

Reply:
@sanjaiganesh I've updated the swagger spec, please take a look.